src/Plugin.php

```diff
@@ -134,19 +139,18 @@ private function loadInstalledPaths()
      */
     private function saveInstalledPaths()
     {
-        // In case we have no installed paths anymore, ensure the config key is deleted, else update it.
-        if (count($this->installedPaths) === 0) {
-            $phpcs = new Process(
-                $this->phpCodeSnifferBin . ' --config-delete installed_paths'
-            );
-        } else {
-            $phpcsInstalledPaths = implode(',', $this->installedPaths);
-            $phpcs = new Process(
-                $this->phpCodeSnifferBin . ' --config-set installed_paths "' . $phpcsInstalledPaths . '"'
-            );
+        // By default we delete the installed paths
+        $arguments = ['--config-delete', 'installed_paths'];
+
+        // This changes in case we do have installed_paths
+        if (count($this->installedPaths) !== 0) {
+            $arguments = ['--config-set', 'installed_paths', implode(',', $this->installedPaths)];
```

Comment:
Is there a specific reason for not using `array_push ($argument, implode(',', $this->installedPaths));` instead of defining the entire array again?